### ISSUE TITLE
Manual "merge request"

### ISSUE THREAD

**Issue by qwazix** [opensourcedesign/fonts#1]
I already have a quite extensive list of free fonts here: https://github.com/qwazix/fonts

Since I cannot create a normal pull request due to my repo not being a fork, I'm requesting permission to just merge the two repos manually. My repo contains a text file with links to fonts that are not open source but gratis. I will remove the list before merging in order to avoid confusion.

**Comment by simonv3** [comment]
I'm not quite sure what you mean by merging the repos manually - you could just set this directory as an other source and push to it. Since there's nothing else in the repository right now I don't think that should be a problem though.

Let me know if I can help with the move!

**Comment by qwazix** [comment]
Yes that is what I meant. Manually in contrast to doing a GitHub merge request. I'll do that later if nobody objects.
